apps/site/public/types/core/0.2/schema/service-meta.json

```diff
@@ -0,0 +1,59 @@
+{
+  "$id": "https://blockprotocol.org/types/core/0.2/schema/service-meta",
```

Comment:
Do we want to add 
```json
"$schema": "http://json-schema.org/draft-07/schema#",
```
to this file and other `0.2` related files as well?

Reply:
Possibly, I'd opt to change the previous spec as little as we can though